ui/admin/app/routes/scopes/scope/targets.js

```diff
@@ -55,7 +55,7 @@ export default class ScopesScopeTargetsRoute extends Route {
    */
   @action
   @loading
-  @notifyError(({ message }) => message)
+  @notifyError(({ message }) => message, { catch: true })
```

Comment:
I catch here so as to not display any errors twice as the form is now also calling this save and also using `notifyError`.